azure-pipelines.yml

```diff
@@ -1,126 +1,125 @@
-          artifactName: '$(ARTIFACT_NAME)'
-        displayName: Share
+          - task: PublishBuildArtifacts@1
+            inputs:
+              pathtoPublish: "$(Build.ArtifactStagingDirectory)"
+              artifactName: "$(ARTIFACT_NAME)"
+            displayName: Share
 
-- stage: DEV
-  displayName: 'Release to devnet (Azure)'
-  dependsOn: CI
-  condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/main')) # Don't run deployment for PRs against the branch.
-  jobs:
-  - template: ${{ variables.releaseTemplate }}
-    parameters:
-      environment: 'dev'
+  - stage: DEV
+    displayName: "Release to devnet (Azure)"
+    dependsOn: CI
+    condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/main')) # Don't run deployment for PRs against the branch.
+    jobs:
+      - template: ${{ variables.releaseTemplate }}
+        parameters:
+          environment: "dev"
 
-- stage: TEST
-  displayName: 'Release to testnet (Azure)'
-  dependsOn: DEV
-  jobs:
-  - template: ${{ variables.releaseTemplate }}
-    parameters:
-      environment: 'test'
-  - job:
-    displayName: 'Load Testing'
-    steps:
-      - task: k6-load-test@0
-        inputs:
-          filename: 'test/k6/ci/status/get-status.js'
-          args: '-e api_base_url=https://app-opdex-test-api-staging.azurewebsites.net'
-        displayName: 'GET /status'
-    dependsOn: 'test' # template deployment name == parameters.environment
-  - job:
-    displayName: 'ReadMe Docs'
-    steps:
-    - task: CmdLine@2
-      displayName: Sync OpenAPI doc
-      inputs:
-        script: |
-          curl https://test-api.opdex.com/swagger/v1/swagger.json > swagger.json
-          npm install rdme
-          npx rdme openapi --id=61161540149ab80029bab5ca --key=$(README_API_KEY)
-      env:
-        README_API_KEY: $(README_API_KEY)
-    dependsOn: 'test' # template deployment name == parameters.environment
+  - stage: TEST
+    displayName: "Release to testnet (Azure)"
+    dependsOn: DEV
+    jobs:
+      - template: ${{ variables.releaseTemplate }}
+        parameters:
+          environment: "test"
+      - job:
+        displayName: "Load Testing"
+        steps:
+          - task: k6-load-test@0
+            inputs:
+              filename: "test/k6/ci/status/get-status.js"
+              args: "-e api_base_url=https://app-opdex-test-api-staging.azurewebsites.net"
+            displayName: "GET /status"
+        dependsOn: "test" # template deployment name == parameters.environment
+      - job:
+        displayName: "ReadMe Docs"
+        steps:
+          - task: CmdLine@2
+            displayName: Sync OpenAPI doc
+            inputs:
+              script: |
+                npm install rdme
+                npx rdme openapi $(Build.SourcesDirectory)/docs/openapi.yml --id=61161540149ab80029bab5ca --key=$(README_API_KEY)
+            env:
+              README_API_KEY: $(README_API_KEY)
```

Comment:
This could probably be moved up to mainnet now, we can start deploying to it even though it';s not indexing yet.